socorro/processor/cache_manager.py

```diff
@@ -91,15 +93,19 @@ def __init__(self):
         self._generator = None
         self.inotify = None
         self.watch_flags = (
-            flags.ACCESS
-            | flags.CREATE
+            flags.CREATE
             | flags.DELETE
             | flags.DELETE_SELF
             | flags.MODIFY
             | flags.MOVED_FROM
             | flags.MOVED_TO
+            | flags.OPEN
         )
 
+        # If the logging level is DEBUG, then we'll treat that as meaning we should be
+        # in verbose mode and emit additional run information
+        self.is_verbose = settings.CACHE_MANAGER_LOGGING_LEVEL == "DEBUG"
```

Comment:
This makes it easier to keep the debugging output, but minimize the work when it's not being emitted.